paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletScreen.kt

```diff
@@ -160,7 +188,24 @@ internal fun WalletBody(
                     onUpdateClicked = onUpdateClicked,
                     onLogoutClicked = onLogoutClicked,
                     onSetDefaultClicked = onSetDefaultClicked,
-                    onAddNewPaymentMethodClicked = onAddNewPaymentMethodClicked,
+                    onAddNewPaymentMethodClicked = {
+                        if (state.addPaymentMethodOptions.size == 1) {
+                            onAddPaymentMethodOptionClicked(state.addPaymentMethodOptions[0])
+                        } else {
```

Comment:
guessing we don't want to show the modal if there's just one PM option, is that the behavior across platforms?